src/app/backend/handler/apihandler.go

```diff
@@ -1412,21 +1415,31 @@ func (apiHandler *APIHandler) handlePutResource(
 
 func (apiHandler *APIHandler) handleDeleteResource(
```

Comment:
I'd move clients creation to the beginning of function since it's very cheap.

Then use something like
```golang
switch kind {
    case ResourceKindJob:
      err = job.DeleteJob(k8sClient, namespace, name)
    default:
      err = verber.Delete(kind, ok, namespace, name)
}

if err != nil {
   // handle it
}

response.WriteHeader(http.StatusOK)
```

It will decrease complexity.